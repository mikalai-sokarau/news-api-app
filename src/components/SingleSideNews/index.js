import React from 'react';

const SingleSideNews = ({ url, title }) => (
  <div>
    <div className="divider" />
    <a href={url} target="_blank">
      <div className="section">
        <h6>{title}</h6>
      </div>
    </a>
  </div>
);

export default SingleSideNews;
