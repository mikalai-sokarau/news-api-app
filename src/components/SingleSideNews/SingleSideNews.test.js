import React from 'react';
import { shallow } from 'enzyme';
import SingleSideNews from '../SingleSideNews';

describe('<SingleSideNews/>', () => {
  test('render SingleSideNews', () => {
    const wrapper = shallow(<SingleSideNews />);
    expect(wrapper).toMatchSnapshot();
  });
});
