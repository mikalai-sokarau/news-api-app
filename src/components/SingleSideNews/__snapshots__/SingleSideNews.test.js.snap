// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SingleSideNews/> render SingleSideNews 1`] = `
<div>
  <div
    className="divider"
  />
  <a
    target="_blank"
  >
    <div
      className="section"
    >
      <h6 />
    </div>
  </a>
</div>
`;
